tests/test_buffer.py

```diff
@@ -1,5 +1,5 @@
 import pytest
```

Comment:
There isn't currently a test covering the case where you get two buffers with different keys, and ensure they are different buffers. You can add that case if you are up to it, the trickier case of getting two separate buffers for the same key is at least tested.

Reply:
Added